en/computer_vision/visual_inertial_odometry.md

```diff
@@ -90,7 +90,7 @@ The following parameters must be set to use external position information with E
 
 Parameter | Setting for External Position Estimation
 --- | ---
-[EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) | Set *vision position fusion*, *vision velocity fusion*, *vision yaw fusion* and *external vision rotation* according to your desired fusion model.
+[EKF2_EV_CTRL](../advanced_config/parameter_reference.md#EKF2_EV_CTRL) | Set *horizontal position fusion*, *vertical vision fusion*, *velocity fusion*, *yaw fusion* and *external vision rotation* according to your desired fusion model.
```

Comment:
Same as above - how do I fix up the last point "external vision rotation".

Reply:
remove "external vision rotation"

Reply:
```suggestion
[EKF2_EV_CTRL](../advanced_config/parameter_reference.md#EKF2_EV_CTRL) | Set *horizontal position fusion*, *vertical vision fusion*, *velocity fusion*, and *yaw fusion* according to your desired fusion model.
```